src/recognition/recognition.controller.ts

```diff
@@ -14,23 +17,15 @@ export class RecognitionController {
         return this.recs.findAll();
     }
     @Post('create')
-        create(@Body() recognition: Recognition): Promise<Recognition>{
-            return this.recs.createRec(recognition);
-        }
-    
+    create(@Body() createRecDto: Recognition): Promise<Recognition>{
```

Comment:
Is there a reason it was changed from recognition back to createRecDto here?

Reply:
No, I think I was just on an older branch and missed that with the other changes. I'll update it with the other changes for the submodule compatibility